docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,108 @@
+# Create Flink job clusters with Helm Chart
+
+## Overview
+
+This Helm Chart is an addition to the [existing](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/config/samples) way of deploying Flink job clusters.
+
+## Prerequisites
+
+
+- Fink Operator Image Version:  ```gcr.io/flink-operator/flink-operator:v1beta1-6``` follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator) to deploy the Operator
+- Flink Image Version: ```flink:1.9.3``` or ```flink:latest```
+- Helm Version 2.x or 3.x
+
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Clone the repository to your local machine, which has access to your running kubernetes cluster.
+  ```bash
+  git clone https://github.com/GoogleCloudPlatform/flink-on-k8s-operator.git
+  ```
+2. Navigate to the following folder: ```/flink-on-k8s-operator/helm-chart```
+
+3. Use the following command to install the Flink job cluster chart:
+  ```bash
+  helm template --namespace example-namespace flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml |kubectl apply -f -
```

Comment:
Seems the `--namespace` flag doesn't work, the cluster is always deployed to the default namespace.

Reply:
true, the --namespace flag should be used after the kubectl apply, basically like this:
```helm template  flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml | kubectl apply --namespace foo -f - ```

Reply:
Actually, I will change the documentation to deploy the cluter without helm template, since it has a disadvantage of not making use of the release feature that helm includes. Using the command ```helm --namespace=foo flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml``` is the better option.